system/Common.php

```diff
@@ -1264,3 +1264,24 @@ function trait_uses_recursive($trait)
         return $traits;
     }
 }
+
+if (! function_exists('request_is')) {
+    /**
+     * Check if a request meets the desired type
+     *
+     * @param string $type
+     *
+     * @return boolean
+     */
+    function request_is(string $type): bool
+    {
+        $request = Services::request(null, true);
```

Comment:
Please use the helper for acquiring services:
```suggestion
        $request = service('request');
```